packages/ui-react/src/containers/Profiles/Form.tsx

```diff
@@ -74,14 +73,13 @@ const Form = ({ initialValues, formHandler, selectedAvatar, showCancelButton = t
         <div className={profileStyles.formFields}>
           {errors.form ? <FormFeedback variant="error">{errors.form}</FormFeedback> : null}
           {submitting && <LoadingOverlay />}
-          <h2 className={profileStyles.nameHeading}>{t('name')}</h2>
```

Comment:
This one seems to be removed completely. Is that on purpose?

Reply:
Yes. It was an arbitrary "Name" on the page. You can see it for your self if you visit your current profile edit page on https://svod-inplayer-acc.demo.videodock.com/